documented-test/src/attr/docs_const.rs

```diff
@@ -29,6 +29,20 @@ fn multiple_docs_work() {
 mod test_customise {
     use documented::docs_const;
 
+    #[test]
+    fn custom_visibility_works() {
+        mod class {
+            use documented::docs_const;
+
+            #[docs_const(vis = pub)]
+            #[allow(dead_code)]
+            /// Arjun!
+            trait RandomStudent {}
+        }
+
+        assert_eq!(class::RANDOMSTUDENT_DOCS, "Arjun!");
```

Comment:
I think it may be prudent to use a proper case conversion library like [convert_case](https://crates.io/crates/convert_case), just so that things like `PascalCase` or `camelCase` (which I don't think is used in Rust at the moment, but it doesn't hurt) are correctly handled.